src/App.js

```diff
@@ -1,31 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/calculator';
 import Navbar from './components/Navbar';
 import Quote from './components/Qoute';
 import HomePage from './components/Home';
 
 function App() {
-  let component;
-  switch (window.location.pathname) {
-    case '/Home':
-      component = <HomePage />;
-      break;
-
-    case '/Calculator':
-      component = <Calculator />;
-      break;
-
-    case '/Qoute':
-      component = <Quote />;
-      break;
-
-    default:
-      component = <HomePage />;
-      break;
-  }
   return (
     <>
       <Navbar />
-      {component}
+      <div className="container">
+        <Routes>
+          <Route path="/Home" element={<HomePage />} />
```

Comment:
- [ ] Kindly update the link to your HomePage, the path is currently not correct, update it to `HomePage` 👍🏿

Reply:
The file containing the homepage component in my work is the Home.js so i think the path is correct that way

Reply:
well resolved! ✔️